src/account-type-utils.js

```diff
@@ -0,0 +1,65 @@
+const validator = require('validator')
+const _ = require('lodash')
+
+/*
+ * Returns the account type based on the user's email address
+ * to use as a Mixpanel property on the user profile.
+ *
+ */
+
+const userTypes = {
+  BroadEmployee: 'Broad Employee',
+  VerilyEmployee: 'Verily Employee',
+  IndependentUser: 'Independent User',
+  EnterpriseUser: 'Enterprise User',
+  ServiceAccountUser: 'Service Account User',
+  EducationalInstituteUser: 'Educational Institute User',
+  NonProfitUser: 'Non-profit User',
+  GovUser: 'Government User',
+  Other: 'Other'
+}
+
+const generalTld = [
```

Comment:
Probably want to stick to the real terms unless there is a reason to deviate.
```suggestion
const topLevelDomains = [
```

Reply:
Scrapped this entirely.